bitnami/dataplatform-bp2/README.md

```diff
@@ -0,0 +1,257 @@
+| `wavefront.collector.resources.limits`     | The resources limits for the collector container                                                                       | `{}`                                                    |
+| `wavefront.collector.resources.requests`   | The requested resources for the collector container                                                                    | Collectors pods resource requests for optimal resource usage size                       |
+| `wavefront.proxy.resources.limits`         | The resources limits for the proxy container                                                                           | `{}`                                                    |
+| `wavefront.proxy.resources.requests`       | The requested resources for the proxy container                                                                        | Proxy pods resource resource requests for optimal resource usage size                                |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+
+```console
+$ helm install my-release \
+  --set kafka.replicaCount=3 \
+  bitnami/dataplatform-bp2
+```
+
+The above command deploys the data platform with Kafka with 3 nodes (replicas).
+
+In case you need to deploy the data platform with Wavefront Observability Framework for all the applications (Kafka/Spark/Elasticsearch/Logstash) in the data platform, you can specify the 'enabled' parameter using the `--set <component>.metrics.enabled=true` argument to `helm install`. For Example,
+
+```console
+$ helm install my-release bitnami/dataplatform-bp2 \
+    --set kafka.metrics.kafka.enabled=true \
+    --set kafka.metrics.jmx.enabled=true \
+    --set spark.metrics.enabled=true \
+    --set elasticsearch.metrics.enabled=true \
+    --set logstash.metrics.enabled=true \
+    --set wavefront.enabled=true \
+    --set wavefront.clusterName=<K8s-CLUSTER-NAME> \
+    --set wavefront.wavefront.url=https://<YOUR_CLUSTER>.wavefront.com \
+    --set wavefront.wavefront.token=<YOUR_API_TOKEN>
+```
+
+The above command deploys the data platform with the Wavefront Observability Framework.
+
+Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install my-release -f values.yaml bitnami/dataplatform-bp2
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
+
+### [Rolling VS Immutable tags](https://docs.bitnami.com/containers/how-to/understand-rolling-tags-containers/)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
```

Comment:
I would expect more information in this section.

Reply:
attending to the above comment, I think it is fine

Reply:
I don't quite agree. You should (at least) refer to the other charts docs since this chart install a lot of stufg and you're not explaining how to configure it.